apps/api/src/app/integrations/e2e/get-decrypted-integrations.e2e.ts

```diff
@@ -3,13 +3,20 @@ import { expect } from 'chai';
 import { ChannelTypeEnum, EmailProviderIdEnum } from '@novu/shared';
 import { IntegrationRepository } from '@novu/dal';
 
+const IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
```

Comment:
```suggestion
const ORIGINAL_IS_MULTI_PROVIDER_CONFIGURATION_ENABLED = process.env.IS_MULTI_PROVIDER_CONFIGURATION_ENABLED;
```
Suggestion. I use this _trick_ to clearly state why I am doing this with the environment variables in my test suites.